dotcom-rendering/src/lib/scheduler.ts

```diff
@@ -73,43 +67,30 @@ const queue: Record<
 };
 
 /**
- * Standardised console logging.
+ * Checks whether there's spare task-running capacity.
  */
-const log = (message: string) =>
-	libsLog('openJournalism', `🧑‍💻 Scheduler ${message}`);
+function atConcurrencyLimit() {
+	return RUNNING_TASK_COUNT === CONCURRENCY_COUNT;
+}
```

Comment:
What happens if you dynamically `setSchedulerConcurrency` to a much smaller number than RUNNING_TASK_COUNT?

```diff
-	return RUNNING_TASK_COUNT === CONCURRENCY_COUNT;
+	return RUNNING_TASK_COUNT >= CONCURRENCY_COUNT;
```

Reply:
ooh eyah, that's a good catch

Reply:
we shoudl add a test to catch it too